app/models/settings/user_experience.rb

```diff
@@ -4,12 +4,18 @@ class UserExperience < Base
     self.table_name = :settings_user_experiences
 
     HEX_COLOR_REGEX = /\A#(\h{6}|\h{3})\z/
+    FEED_STRATEGIES = %w[basic large_forem_experimental].freeze
+    FEED_STYLES = %w[basic rich compact].freeze
```

Comment:
This seemed like the right way to keep the options maintainable. I didn't make constants out of `default_font` and `default_locale` though since the values we want to display are different than the values we set in the DB.